.github/contributing.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to ZDS Token Issuer
+
+## Reporting issues
+
+Please report issues to the [ZDS Token Issuer][token-issuer-issues] repository.
+
+We manage issues/bugs/userstories centralized so that they can be prioritized
+by the product owner and planned into the sprint. There are templates available
+for new user stories or bugs.
+
+## Implementing features
+
+We welcome enthusiasm for new features!
+
+However, to prevent dissapointment, we recommend you to:
+
+* create a user story in [ZDS Token Issuer][token-issuer-issues] - it's possible that
+  the conclusion is that we don't want to support that feature (for the time
+  being)
+
+**Style guides**
+
+* we use `isort` to make sure imports are consistently sorted (config in `setup.cfg`)
```

Comment:
I would also expect `black` to be mentioned here, since it's part of the CI code quality checks